src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -993,13 +1013,24 @@ void readAnalyze(TrackPointer track, double sampleRate, int timingOffset, bool i
         }
     }
 
-    if (cueLoadPosition < kLongestPosition) {
-        track->setCuePoint(CuePosition(cueLoadPosition));
+    std::sort(memoryCues.begin(), memoryCues.end(), [](const memory_cue_t& a, const memory_cue_t& b) -> bool {
+        return a.position < b.position;
+    });
+
+    if (memoryCues.size() > 0) {
```

Comment:
Just minor nitpicking: Move the call to std::sort() into this block. Sorting an empty list is pointless and picking the first element right after sorting is more obvious.